repository/build.gradle

```diff
@@ -1,25 +1,29 @@
 jar {
-    baseName = 'definition-designer-repository'
+    enabled = true
+    manifest {
+        attributes 'Implementation-Title': project.name,
+                'Implementation-Version': project.version
+    }
+    archiveName = 'definition-designer-repository'
 }
 
-bootRepackage {
+bootJar {
     enabled = false
 }
 
 dependencies {
     compile("org.springframework.boot:spring-boot-starter-data-jpa") {
         exclude group: 'org.apache.tomcat', module: 'tomcat-jdbc'
     }
-    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '0.6.0'
-    compile "commons-beanutils:commons-beanutils"
+    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '1.0.4'
+    compile "commons-beanutils:commons-beanutils:1.9.3"
```

Comment:
I don't think we need to say `1.9.31 here

Reply:
throws the following error when we omit the version there
```* What went wrong:
Could not determine the dependencies of task ':bootJar'.
> Could not resolve all task dependencies for configuration ':runtimeClasspath'.
   > Could not find commons-beanutils:commons-beanutils:.
     Required by:
         project : > project :application > project :rest-api > project :domain > project :repository
```